src/main/resources/chains/eth.conf

```diff
@@ -37,6 +37,10 @@
   # https://ecips.ethereumclassic.org/ECIPs/ecip-1088
   phoenix-block-number = "1000000000000000000"
 
+  # Petersburg fork block number (ETH only)
+  # https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1716.md
+  petersburg-block-number = "7280000"
```

Comment:
hmm mayble as we are adding it, we should also change constantinople fork block number in config to `7280000` ? (https://media.consensys.net/the-constantinople-hard-fork-what-you-need-to-know-d438a91dec3f)